app/build.gradle

```diff
@@ -50,6 +50,7 @@ dependencies {
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
+      'androidx.legacy:legacy-support-v13:1.0.0'
```

Comment:
Why do we need this dep?

Reply:
it was added for some other purpose. It is not needed now so removed it